test/models/submission_information_package_test.rb

```diff
@@ -146,6 +146,19 @@ class SubmissionInformationPackageTest < ActiveSupport::TestCase
     assert_not sip.valid?
   end
 
+  test 'a SIP is invalid if its thesis has accession number' do
```

Comment:
The test looks good, the name is backwards. I think it means to be `a SIP is invalid if its thesis has NO accession number`